letsencrypt/client/client.py

```diff
@@ -153,25 +157,34 @@ def save_certificate(self, certificate_msg, cert_path, chain_path):
         :raises IOError: If unable to find room to write the cert files
 
         """
+        # try finally close
         cert_chain_abspath = None
-        cert_fd, cert_file = le_util.unique_file(cert_path, 0o644)
-        cert_fd.write(certificate_msg.certificate.as_pem())
-        cert_fd.close()
+        cert_file, act_cert_path = le_util.unique_file(cert_path, 0o644)
+        # TODO: Except
+        try:
+            cert_file.write(certr.body.as_pem())
```

Comment:
`certr.body.as_pem()` should be moved outside the `try` block; can you fix the TODO above and except a list possible exception classes?